irr/src/CMakeLists.txt

```diff
@@ -535,7 +525,7 @@ target_link_libraries(IrrlichtMt PRIVATE
 	"$<$<BOOL:${OPENGLES_DIRECT_LINK}>:${OPENGLES_LIBRARY}>"
 	${EGL_LIBRARY}
 
-	"$<$<PLATFORM_ID:Android>:-landroid -llog>"
+	"$<$<PLATFORM_ID:Android>:-landroid -llog -lGLESv2 -lGLESv1_CM -lOpenSLES>"
```

Comment:
this should not be needed
except `OpenSLES` which could be a transitive dependency from SDL?

Reply:
All of them are transitive dependencies from SDL. Removing them results in lots of linker errors.

Reply:
```suggestion
	# incl. transitive SDL2 dependencies for static linking
	"$<$<PLATFORM_ID:Android>:-landroid -llog -lGLESv2 -lGLESv1_CM -lOpenSLES>"
```